test_phage_reference/1_build_reference_transcriptome.ipynb

```diff
@@ -41,19 +41,383 @@
+      "[Building BooPHF]  99.9 %   elapsed:   0 min 0  sec   remaining:   0 min 0  sec\n",
+      "Bitarray        29598784  bits (100.00 %)   (array + ranks )\n",
+      "final hash             0  bits (0.00 %) (nb in final hash 0)\n",
+      "\u001b[00m[2020-12-22 10:50:53.472] [puff::index::jointLog] [info] mphf size = 3.52845 MB\n",
+      "\u001b[00m\u001b[00m[2020-12-22 10:50:53.477] [puff::index::jointLog] [info] chunk size = 2,945,326\n",
+      "\u001b[00m\u001b[00m[2020-12-22 10:50:53.478] [puff::index::jointLog] [info] chunk 0 = [0, 2,945,326)\n",
+      "\u001b[00m\u001b[00m[2020-12-22 10:50:53.478] [puff::index::jointLog] [info] chunk 1 = [2,945,326, 5,890,622)\n",
+      "\u001b[00m\u001b[00m[2020-12-22 10:50:53.857] [puff::index::jointLog] [info] finished populating pos vector\n",
+      "\u001b[00m\u001b[00m[2020-12-22 10:50:53.857] [puff::index::jointLog] [info] writing index components\n",
+      "\u001b[00m\u001b[00m[2020-12-22 10:50:53.869] [puff::index::jointLog] [info] finished writing dense pufferfish index\n",
+      "\u001b[00m[2020-12-22 10:50:53.986] [jLog] [info] done building index\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Get PA14 index\n",
+    "! salmon index -t $paths.PA14_REF -i $paths.PA14_INDEX"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Version Info: ### PLEASE UPGRADE SALMON ###\n",
+      "### A newer version of salmon with important bug fixes and improvements is available. ####\n",
+      "###\n",
+      "The newest version, available at https://github.com/COMBINE-lab/salmon/releases\n",
+      "contains new features, improvements, and bug fixes; please upgrade at your\n",
+      "earliest convenience.\n",
+      "###\n",
+      "Sign up for the salmon mailing list to hear about new versions, features and updates at:\n",
+      "https://oceangenomics.com/subscribe\n",
+      "###\n",
+      "[2020-12-22 10:50:54.175] [jLog] [warning] The salmon index is being built without any decoy sequences.  It is recommended that decoy sequence (either computed auxiliary decoy sequence or the genome of the organism) be provided during indexing. Further details can be found at https://salmon.readthedocs.io/en/latest/salmon.html#preparing-transcriptome-indices-mapping-based-mode.\n",
+      "[2020-12-22 10:50:54.176] [jLog] [info] building index\n",
+      "out : /home/alexandra/Documents/Data/Core_accessory/phage_index\n",
+      "\u001b[00m[2020-12-22 10:50:54.176] [puff::index::jointLog] [info] Running fixFasta\n",
+      "\u001b[00m\n",
+      "[Step 1 of 4] : counting k-mers\n",
+      "\u001b[35m[2020-12-22 10:50:54.270] [puff::index::jointLog] [warning] Entry with header [NC_028999.1] was longer than 200000 nucleotides.  This is probably a chromosome instead of a transcript.\n",
```

Comment:
This is because for the phage reference, I'm using the phage genomes as opposed to phage genes. I'm not exactly sure if Salmon handles these differently so something that would be worth checking. Thanks